sysconfig/cloudinit.go

```diff
@@ -151,9 +151,13 @@ var (
 	nocloudRestrictYaml = []byte(`datasource_list: [NoCloud]
 datasource:
   NoCloud:
-    fs_label: null`)
+    fs_label: null
+manual_cache_clean: true
+`)
 
-	genericCloudRestrictYamlPattern = `datasource_list: [%s]`
+	genericCloudRestrictYamlPattern = `datasource_list: [%s]
+manual_cache_clean: true
```

Comment:
I'm not sure why we want do this for actual clouds,  the instance id there comes from the cloud itself, no?

Reply:
As discussed this was removed and now `manual_cache_clean` is only used for the NoCloud datasource